site/content/docs/user/ingress.md

```diff
@@ -8,25 +8,29 @@ menu:
 description: |-
   This guide covers setting up [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) on a kind cluster.
 ---
-## Setting Up An Ingress Controller
+## Compatibility：
+This guide applies to [cloud-provider-kind](https://github.com/kubernetes-sigs/cloud-provider-kind) v0.9.0+. For older versions, refer to historical docs.
+
+## Setting Up Ingress
 
 Ingress exposes HTTP and HTTPS routes from outside the cluster to services within the cluster.
 
-1. [Create a cluster](#create-cluster): There are two primary methods to direct external traffic to Services inside the cluster:
-    1. using a [LoadBalancer].
-    2. leverage KIND's `extraPortMapping` config option when creating a cluster to forward ports from the host.
+[Create a cluster](#create-cluster): There are two primary methods to direct external traffic to Services inside the cluster:
+- using a [LoadBalancer]. 
+- leverage KIND's `extraPortMapping` config option when creating a cluster to forward ports from the host.
+
+> **NOTE**: Since kind v0.9.0, the built-in cloud provider (cloud provider kind) natively supports Ingress. You do not need to deploy a third-party Ingress controller (e.g., Ingress NGINX, Contour) by default.
 
-2. Deploy an Ingress controller, we document [Ingress NGINX](#ingress-nginx) here but other ingresses may work including [Contour](https://projectcontour.io/docs/main/guides/kind/) and Kong, you should follow their docs if you choose to use them.
+For third-party ingress solutions (e.g., Ingress NGINX, Contour), please follow their official documentation.
 
-> **NOTE**: You may also want to consider using [Gateway API](https://gateway-api.sigs.k8s.io/) instead of Ingress.
-> Gateway API has an [Ingress migration guide](https://gateway-api.sigs.k8s.io/guides/migrating-from-ingress/).
+> **NOTE**: Since kind v0.9.0, Gateway API is also natively supported (along with Ingress). You may also want to consider using [Gateway API](https://gateway-api.sigs.k8s.io/) instead of Ingress.
+> See the official [Ingress migration guide](https://gateway-api.sigs.k8s.io/guides/migrating-from-ingress/) for details.
 
 ### Create Cluster
 
 #### Option 1: LoadBalancer
 
-Create a kind cluster and run [Cloud Provider KIND]
-to enable the loadbalancer controller which ingress-nginx will use through the loadbalancer API.
+Create a kind cluster. The built-in cloud provider (cloud provider kind, enabled by default since v0.9.0) will automatically enable the LoadBalancer controller for Ingress to use.
```

Comment:
```suggestion
Create a kind cluster and run [Cloud Provider KIND] to enable the loadbalancer controller which ingress controllers will use through the loadbalancer API. Since version 0.9.0 [Cloud Provider KIND] also supports Ingress implementation```

Reply:
remove Option 1 and Option 2, we do have only one option now